### PR TITLE
Update Inputs “Select”

### PR DESCRIPTION
**Pull request by owendodd**
Automatically generated by Netlify CMS

### CODE REVIEW

File: packages/palette-docs/content/docs/elements/inputs/Select.mdx [hunk 1]
```diff
@@ -27,7 +30,7 @@ of user-benefitting product strategy.
   <Serif size="3">Title</Serif>
   <Serif size="2">Short description</Serif>
   <LargeSelect
-    options={[
+    options={\[
```
Comment (oxaudo) [inline packages/palette-docs/content/docs/elements/inputs/Select.mdx]: I think those are stray elements here...

Comment (zephraph) [reply packages/palette-docs/content/docs/elements/inputs/Select.mdx]: Yeah, for some reason the rich text editor mode in netlify-cms adds those. 

Need to file an upstream issue (or possibly update netlify-cms)

**Comment by artsyit** [comment]
Deploy preview for *artsy-palette* ready!

Built with commit f7d1b0c3ee366d3134fb7b255684bff768bcc5bd

https://deploy-preview-469--artsy-palette.netlify.com

**Comment by artsyit** [comment]
:rocket: PR was released in v4.14.17 :rocket:

**Pull request merged**